plugins/poll/assets/javascripts/components/poll-breakdown-chart.js.es6

```diff
@@ -0,0 +1,175 @@
+import I18n from "I18n";
+import Component from "@ember/component";
+import { htmlSafe } from "@ember/template";
+import discourseComputed from "discourse-common/utils/decorators";
+import { PIE_CHART_TYPE } from "../controllers/poll-ui-builder";
+import { getColors } from "../lib/chart-colors";
+
+// args: options, group, displayMode, highlightedOption, setHighlightedOption
```

Comment:
One thing you could do here is put them in the declaration with `null` as the value. Javascript likes this actually, because it declares the "shape" of the object which can have better performance.